packages/server/src/sdk/tests/attachments.spec.ts

```diff
@@ -1,4 +1,12 @@
-import newid from "../../db/newid"
+import TestConfig from "../../tests/utilities/TestConfiguration"
```

Comment:
This test is a bit unrelated but was failing for me so I fixed it. It was hardcoding table names in a way that didn't play nice if the test was re-run with `REUSE_CONTAINERS=true` set. I've modernised it and made it play nice.

Reply:
It was, yeah. Worked fine on a fresh DB, but running a 2nd time (either through `REUSE_CONTAINERS` or `--watch`) it would fail.